teleport-server/instance.tf

```diff
@@ -0,0 +1,57 @@
+resource "aws_instance" "teleport_instance" {
+  ami                         = "${var.ami_id}"
+  instance_type               = "${var.instance_type}"
+  key_name                    = "${var.key_name}"
+  iam_instance_profile        = "${aws_iam_instance_profile.profile.id}"
+  vpc_security_group_ids      = ["${aws_security_group.teleport_server.id}"]
+  subnet_id                   = "${var.subnet_id}"
+  disable_api_termination     = "false"
+  ebs_optimized               = "${contains(var.ebs_optimized_list, var.instance_type)}"
+  associate_public_ip_address = "true"
+  user_data                   = "${data.template_cloudinit_config.teleport.rendered}"
+
+  root_block_device {
+    volume_type           = "${var.root_vl_type}"
+    volume_size           = "${var.root_vl_size}"
+    delete_on_termination = "${var.root_vl_delete}"
+  }
+
+  tags {
```

Comment:
Please use upper case for the first letter, same as in other modules. 
https://github.com/skyscrapers/terraform-instances/blob/master/instance/main.tf#L56

Reply:
Done